tutorials/themes/tutorials-theme/static/searchtools.js

```diff
@@ -416,17 +416,17 @@ var Search = {
       if (query.includes("filterTutorials")) { // don't search docs if we are just looking for tutorials
         var docResponse = null;
         var docResult = '';
-        $.get('https://readthedocs.org/api/v2/docsearch/?q=' + window.location.search.split('=')[1] + '&project=astropy&version=stable&language=en', function(response){
+        $.get('https://docs.readthedocs.io/_/api/v2/search/?q=' + window.location.search.split('=')[1] + '&project=astropy&version=stable', function(response){
```

Comment:
The new endpoint should be https://docs.astropy.org/_/api/v2/search/?q=test&project=astropy&version=stable

Reply:
oh, I think you can use the other endpoint https://readthedocs.org/api/v2/search/?q=test&project=astropy&version=stable let me know if that one works

Reply:
I still get

> Access to XMLHttpRequest at 'https://readthedocs.org/api/v2/search/?q=test&project=astropy&version=stable' from origin 'http://learn.astropy.org' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

:/

Reply:
hmm, not sure how this was working before.

https://github.com/readthedocs/readthedocs.org/blob/12598254c86e954f19c7402395d9e402f9a2aaaa/readthedocs/core/signals.py#L33-L82

Looks like you can get around by creating a custom domain with "learn.astropy.org", there is no need to add the CNAME, this is so rtd knows this is a known domain.

Reply:
I mean, creating the custom domain the astropy project.

Reply:
I'm not sure if it *was* working before.  Presumably it was at some point. But even with the current version of the site using the old/undocumented search it also gets CORS errors, so it isn't working currently either.

Reply:
@stsewd I see.  It turns out learn.astropy.org is not even built/hosted by RTD.  I assumed it was, but apparently it is using a CI service to build the site and push it to gh-pages.  Seeing that code helps, though it's still not exactly clear to me how I can add learn.astropy.org as a known domain without hosting these docs on RTD (which I'm not opposed to, but someone who was not me apparently made the decision at one point not to).

Reply:
Oh, I found the settings on RTD for adding an additional custom domain.  I see your point now about adding it as a custom domain without actually adding a CNAME entry.  Hopefully that should work (it does not yet, but it still shows "SSL certificate status: pending_validation" on https://readthedocs.org/dashboard/astropy/domains/10700/edit/

Reply:
It appears to be due to this, but in this case RTD wouldn't be issuing an SSL cert on behalf of the site: https://docs.readthedocs.io/en/stable/custom_domains.html#certificate-authority-authorization

Reply:
Although looking again at the code you linked, for the CORS access it doesn't seem to care about certificate status :/

Reply:
> someone who was not me apparently made the decision at one point not to

@adrn , @eblur , or @kelle , do you remember why this decision was made? Can you please help @embray move this forward? Thanks.

Reply:
> Although looking again at the code you linked, for the CORS access it doesn't seem to care about certificate status

Yeah, you only need to add the domain, the status or the CNAME don't matter here. Let me know if that fixed the CORS problem.